data/scripts/clean.py

```diff
@@ -0,0 +1,6 @@
+data = open('out/data','r').read().split('\n')
+for i,line in enumerate(data):
+    if len(line) > 5:
+        data[i] = line.replace('"','',2)
```

Comment:
if this is for turning `{ "owo": 2 }` to `{ owo: 2 }`, u can delete it because quotes vs no quotes are both allowed in typescript and mean the same thing